File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        ans = []
        for i in range (0,n):
            ans.append(nums[i])
        for i in range (0,n):
            ans.append(nums[i])

        return ans
        

File: Data Structures & Algorithms/majority-element-ii/submission-0.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        std::vector<int> ans;
        int n=nums.size();
        unordered_map<int,int> mpp;
        for(int i=0;i<n;i++){
            mpp[nums[i]]++;
            if(mpp[nums[i]]==n/3+1){
                ans.push_back(nums[i]);
            }
        }
        return ans;

    }
};